src/config/index.ts

```diff
@@ -0,0 +1,5 @@
+export default {
+  port: 3000,
+  dbUrl:
+    'postgres://hexaunhz:QPLbFw6DkBV9-5nSv98k7dFeUeWYC48R@snuffleupagus.db.elephantsql.com/hexaunhz',
```

Comment:
it's better to get dbUrl from process.env and set process.env variables via .env for example. In this case you do not commit credentials to your resources

Reply:
fixed. moved these configs to env file